scripts/benchmarks/run_yolox_tiny_single.yml

```diff
@@ -0,0 +1,6 @@
+nodes:
+- input.recorded:
+    input_dir: images/benchmarking/single
```

Comment:
I'm thinking - to avoid committing in the .mp4 files by accident, it may be better to include the benchmarking videos in a folder that's not called `images` and include that folder name in our `.gitignore`.

Reply:
This will be influenced by the outcome from the above comment.